lib/lib.go

```diff
@@ -419,8 +419,13 @@ func (p *Podium) GetMember(ctx context.Context, leaderboard, memberID string) (*
 }
 
 // GetMembersAroundMember returns the members around the given memberID
-func (p *Podium) GetMembersAroundMember(ctx context.Context, leaderboard, memberID string, pageSize int) (*MemberList, error) {
-	route := p.buildGetMembersAroundMemberURL(leaderboard, memberID, pageSize)
+func (p *Podium) GetMembersAroundMember(ctx context.Context, leaderboard, memberID string, pageSize int, lastIfNotFound bool, order ...string) (*MemberList, error) {
+	var o = "desc"
+	if len(order) > 0 {
+		o = order[0]
+	}
+
```

Comment:
nit: these spaces aren't really needed

Reply:
What spaces exactly? I just copied those lines from the GetMembersInLeaderboard method.

Reply:
empty lines*

Reply:
I can do that. It won't be consistent with the rest, though.

Reply:
it's up to you. Just nitpicking :) In go it's idiomatic to not have empty lines inside functions most of the time.

Reply:
I decided to maintain consistency with the rest of the file. Removed an empty line to better match the style employed so far. I will keep that in mind for the future though!